### PR TITLE
Import external tx to active record

### PR DESCRIPTION
**Pull request by Yamaguchi**
This PR is associated with #26 
This makes it possible to external transaction created without glueby library into active record database.

### CODE REVIEW

File: lib/tasks/glueby/contract/wallet_adapter.rake [hunk 1]
```diff
@@ -0,0 +1,42 @@
+module Glueby
+  module Contract
+    module Task
+      module WalletAdapter
+        def import_block(block_hash)
+          block = Glueby::Internal::RPC.client.getblock(block_hash, 0)
+          block = Tapyrus::Block.parse_from_payload(block.htb)
+          block.transactions.each { |tx| import_tx(tx.txid) }
```
Comment (azuchi) [inline lib/tasks/glueby/contract/wallet_adapter.rake:8]: Do you need to set (Database) transaction block to roll back in the event of an unexpected error?

File: lib/glueby/internal/wallet/active_record/key.rb [hunk 2]
```diff
@@ -27,6 +27,38 @@ def address
...
+          # Return Glueby::Internal::Wallet::AR::Key object for input.
+          # If utxo spent by the specified input is colored output, key is found by corresponding `uncolored` script.
+          #
+          # @param [Tapyrus::TxIn] input
+          # @return [Glueby::Internal::Wallet::AR::Key] key for input
+          def self.key_for_input(input)
+            out_point = input.out_point
+            utxo = Utxo.find_by(txid: out_point.txid, index: out_point.index)
+            return unless utxo
+            script_pubkey = Tapyrus::Script.parse_from_payload(utxo.script_pubkey.htb)
```
Comment (azuchi) [inline lib/glueby/internal/wallet/active_record/key.rb:39]: You can call `key_for_output` using this `script_pubkey`. As results, rest of following code can be removed.

**Review by azuchi — CHANGES_REQUESTED** [review]

File: lib/tasks/glueby/contract/wallet_adapter.rake (hunk 1 above)
Comment (Yamaguchi) [reply lib/tasks/glueby/contract/wallet_adapter.rake:8]: Yes. Fixed at 93a657d.

File: lib/glueby/internal/wallet/active_record/key.rb (hunk 2 above)
Comment (Yamaguchi) [reply lib/glueby/internal/wallet/active_record/key.rb:39]: I added a new method Key.key_for_script (10abdae)

**Review by azuchi — APPROVED** [review]

**Pull request merged**